src/ui/components/NetworkMonitor/RequestRow.tsx

```diff
@@ -38,8 +44,21 @@ export const RequestRow = ({
     prevIsSelectedRef.current = isSelected;
   }, [isSelected]);
 
+  const sessionId = useAppSelector(getSessionId)!;
+
   const { contextMenu, onContextMenu } = useNetworkContextMenu(row);
 
+  const onNetworkContextMenu = useCallback(
+    async (event: any) => {
+      onContextMenu(event);
+      if (data.hasRequestBody) {
+        await client.Network.getRequestBody({ id: data.id, range: { end: 5e9 } }, sessionId);
+        dispath(enableCopyCUrl(data.id));
+      }
```

Comment:
This looks unexpected to me.

Why would we need to eagerly fetch this? Why not just fetch it if/when the button is clicked?